website/frontend/ErrorBoundary.jsx

```
@@ -0,0 +1,43 @@
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <div className="text-center p-6">
+          <h1 className="text-2xl font-bold text-red-600">Something went wrong.</h1>
+          <p className="text-gray-700 mt-4">An error occurred, and we are working to fix it.</p>
+          {process.env.NODE_ENV === 'development' && (
+            <details className="whitespace-pre-wrap text-sm text-left mt-4">
+              <summary className="cursor-pointer">Error Details</summary>
+              {this.state.error && this.state.error.toString()}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)